src/pathwayvis/components/map/map.component.ts

```diff
@@ -386,16 +403,17 @@ class MapComponentCtrl {
       d3.selectAll('#reactions > .reaction').style('filter', null);
       const model = this._mapOptions.getDataModel();
       this._builder.load_model(model);
-      const reactionsToHighlight = model.notes.changes
-        .measured.reactions.map((reaction) => reaction.id);
-      reactionsToHighlight.forEach((reactionId) => {
-        this._builder.map.bigg_index
-          .getAll(reactionId)
-          .forEach(({reaction_id}) => {
-            d3.select(`#r${reaction_id}`)
-              .style("filter", "url(#escher-glow-filter)");
-          });
-      });
+      if (model.notes.changes) {
+        const reactionsToHighlight = model.notes.changes.measured.reactions.map((reaction) => reaction.id);
```

Comment:
Looking at this `reactionsToHighlight` could be inlined, just add a few linebreaks. Thank you!